site/zenodo_rdm/config.py

```diff
@@ -312,3 +313,6 @@
 }
 
 REDIRECTOR_RULES.update(EXPORT_REDIRECTS)
+
+# Permissions
+RDM_PERMISSION_POLICY = ZenodoRDMRecordPermissionPolicy
```

Comment:
Question: Why do we need to set here the `RDM_PERMISSION_POLICY` when it was already set in the `invenio.cfg`?